tty.go

```diff
@@ -55,9 +55,12 @@ func createTty(p *libcontainer.Process, rootuid int, consolePath string) (*tty,
 	go io.Copy(console, os.Stdin)
 	go io.Copy(os.Stdout, console)
 
-	state, err := term.SetRawTerminal(os.Stdin.Fd())
-	if err != nil {
-		return nil, fmt.Errorf("failed to set the terminal from the stdin: %v", err)
+	var state *term.State
+	if term.IsTerminal(os.Stdin.Fd()) {
```

Comment:
@ashahab-altiscale  No, it's different, `create` is whether to create a `tty` to container or not. `create` is `true`, but the `stdin` could not be a terminal.